PULL-REQUESTS.md

```diff
@@ -0,0 +1,86 @@
+# Pull Request Checklist
+
+## Beginning the Work
+Good Pull Requests start well before you write your first line of code. Be prepared to craft discreet, focused and intentioned PR's by following a few simple guidelines:
+
+1. Create a __new__ branch
+1. Select a branch name that matches your company's guidelines
+1. In the absence of guidelines, use a consistent, meaningful branch naming convention. Consider using the story or task identifier in the branch name.
+1. Keep the work to a single feature / story / task to limit the scope of the upcoming PR
+1. Use meaningful commit messages:
+  * Great: `ABC-123: Updated logic to include the correct coefficient multiplier based on the information provided by @teammate in our grooming session on 10/02`
+  * Good: `Updated logic to include the correct coefficient multiplier`
+  * Poor: `Fixed some things`
+  * Bad:  ` `
+
+## Before Submitting a PR
+Good code reviews begin with you. Make sure you have done everything possible to ensure that the process goes quickly and that the reviewers have the necessary information to complete the review. Now is the time to take care of the easy things so that the reviewers can focus on the code itself:
+
+1. Make sure the code builds / compiles properly, even if you only made a small change
+1. Ensure all Tests in your test suite are passing (if you have a test suite)
+1. Make sure all Linter warnings and errors have been addressed
+1. If you don't have a Linter, ensure your indentation, spacing and formatting is correct and consistent
+1. Make sure you have properly commented your code. If your team uses JSDoc, JavaDoc or something similar, make sure you're compliant
+1. Remove all superfluous logging statements (e.g.: `console.log`, `console.warn`, `System.out.println`)
+1. Remove all commented-out / hidden code unless including an explanation as to why it is being temporarily hidden
+1. If you have copied/pasted code from an online resource, read and review it yourself first. Ensure it meets your standards / requirements.
+1. Make sure there are no unnecessary files changed
+1. Limit your PR's to a single story and/or ticket
+1. Limit the size of your PR's as much as possible
+
+## When Submitting a PR
+Aside from writing good, clean code, completing the tasks you're assigned and then running through a "pre-submit checklist", there are some guidelines for how to submit a Pull Request to ensure things go as smoothly as possible:
+
+1. Make sure to highlight dependencies on other branches/projects/versions (e.g.: requires API v2, requires API branch `foo` be merged)
```

Comment:
Perhaps also add if once the branch gets merged in, if a `yarn install` or `npm install` is necessary

Reply:
This is a great one, it seems to bite me a couple of times a week!